packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -136,11 +137,26 @@ class RefreshIndicator extends StatefulWidget {
   /// Typically a [ListView] or [CustomScrollView].
   final Widget child;
 
-  /// The distance from the child's top or bottom edge to where the refresh
-  /// indicator will settle. During the drag that exposes the refresh indicator,
-  /// its actual displacement may significantly exceed this value.
+  /// The distance from the child's top or bottom [edge offset][edgeOffset] where
```

Comment:
This should be just 'bottom [edgeOffset]' as [edge offset] won't generate a valid link in the docs.

Reply:
Done.